hoodie-common/src/main/java/com/uber/hoodie/common/table/log/HoodieLogFileReader.java

```diff
@@ -49,6 +48,7 @@
 class HoodieLogFileReader implements HoodieLogFormat.Reader {
 
   private static final int DEFAULT_BUFFER_SIZE = 4096;
+  private static final int DEFAULT_LOG_BLOCK_SIZE = 256*1024*1024;
```

Comment:
should the default be this high? this will add to the memory pressure also correct?

Reply:
The 256MB is just read once as a byte [] and then discarded, should be ok I think.